DESCRIPTION

```diff
@@ -1,8 +1,15 @@
 Package: MsExperiment
 Title: Infrastructure for Mass Spectrometry Experiments
-Version: 0.6.1
+Version: 0.99.0
 Description: Infrastructure to store and manage all aspects related to
-   a complete proteomics or metabolomics mass spectrometry experiment.
+   a complete proteomics or metabolomics mass spectrometry (MS) experiment. The
+   MsExperiment package provides light-weight and flexible containers for MS
+   experiments building on the new MS infrastructure provided by the Spectra,
+   QFeatures and related packages. Along with raw data representations, links
+   to original data files and sample annotations, also additional metadata or
+   annotations can be stored within the MsExperiment container. To guarantee
```

Comment:
~~also~~ additional metadata or annotations can *also* be stored

Reply:
Fixed